lazy-pages/src/lib.rs

```diff
@@ -430,6 +432,7 @@ fn init_with_handler<H: UserSignalHandler>(
                 page_sizes,
                 global_names,
                 pages_storage_prefix,
+                program_storage: Box::new(pages_storage),
```

Comment:
Wouldn't it lead to valuable perf decreases?

Reply:
You have no way to make it generic because of TLS